ui.go

```diff
@@ -932,8 +951,8 @@ func (ui *ui) draw(nav *nav) {
 			preview := ui.wins[len(ui.wins)-1]
 
 			if curr.IsDir() {
-				preview.printDir(ui.screen, ui.dirPrev, &context,
-					&dirStyle{colors: ui.styles, icons: ui.icons, previewing: true})
+				style.previewing = true
```

Comment:
Minor nit: I'm not a fan of this refactoring. I prefer as many variables as possible to be immutable, so that the reader doesn't have to pay careful attention to how it might change on them. This makes it seem that there's some important and confusing logic that will use the value of `style` below.

Reply:
If you don't like the original version, I would prefer:

```
previewStyle := dirStyle{colors: style.colors, icons: style.icons, previewing: false}
preview.printDir(..., previewStyle)
```

I'm unsure if Go provides a shorter way to write that.